Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.cpp

```diff
@@ -244,9 +244,18 @@ void WebSWServerToContextConnection::fireBackgroundFetchClickEvent(ServiceWorker
 
 void WebSWServerToContextConnection::terminateWorker(ServiceWorkerIdentifier serviceWorkerIdentifier)
 {
+    protectedConnection()->protectedNetworkProcess()->protectedParentProcessConnection()->send(Messages::NetworkProcessProxy::AcquireServiceWorkerTerminationAssertion { webProcessIdentifier(), serviceWorkerIdentifier }, 0);
```

Comment:
I'll probably do a small refactoring to introduce `sendToParentProcess`.

Reply:
I think I looked at that in the beginning but was concerned about overloading the meaning of `m_processingFunctionalEventCount`. But let me see if I can make that work.